pkg/localstore/localstore.go

```diff
@@ -166,7 +166,7 @@ func New(path string, baseKey []byte, o *Options, logger logging.Logger) (db *DB
 	if db.capacity == 0 {
 		db.capacity = defaultCapacity
 	}
-	db.logger.Infof("db capacity: %v", db.capacity)
+	db.logger.Infof("database capacity in chunks (and in Bytes): %v (%v)", db.capacity, db.capacity*swarm.ChunkSize)
```

Comment:
i think it might be nicer to have a message like:
`database capacity: %d chunks (%d bytes, %d megabytes)`.
counting in bytes is so 90s :)

Reply:
Updated with slight change from what you suggested